### ISSUE TITLE
Change headers to enable CORS

### ISSUE THREAD

**Issue by SuperV1234** [lepture/python-livereload#213]
I would like to set my headers to something like

```
server.setHeader('Access-Control-Allow-Origin', '*')
server.setHeader('Access-Control-Allow-Methods', '*')
```

but it doesn't seem this functionality is exposed anywhere. What can I do to set headers?

**Comment by bmcculley** [comment]
Hi @SuperV1234 this should be addressed with PR #216

**Comment by bmcculley** [comment]
Closing this, new release on pypi addresses this issue.
